frontend/src/App.tsx

```diff
@@ -1,9 +1,12 @@
-import './App.css';
+import '@bcgov/design-tokens/css/variables.css';
+import '@bcgov/bc-sans/css/BC_Sans.css';
```

Comment:
Can we move `App.tsx` to be an `@emotion/styled` component and config it's css there?

Reply:
We could make a style for App, but based on the scenario I guess, when we go to that extent then create a ticket!